test/ComponentTests/PlatformServices.Desktop.Component.Tests/DesktopTestSourceHostTests.cs

```diff
@@ -75,26 +67,25 @@ public void ChildDomainResolutionPathsShouldHaveSearchDirectoriesSpecifiedInRuns
 
         var assemblyResolution = "ComponentTests\\TestProjectForAssemblyResolution.dll";
         var asm = Assembly.LoadFrom(assemblyResolution);
-        var type = asm.GetType("PlatformServices.Desktop.ComponentTests.TestProjectForAssemblyResolution");
+        var type = asm.GetType("PlatformServices.Desktop.ComponentTests.SerializableTypeThatShouldBeLoaded");
```

Comment:
please can you explain, the difference between the two of them

Reply:
Definitely :) Basically we don't need the project we load to be a test (see https://github.com/microsoft/testfx/pull/1252/files#r970936904) so to make it more obvious I decided to rename the type so we don't think it's a test that is missing attributes or something.